### ISSUE TITLE
Problem: pbzip2 compression will fail on reingest

### ISSUE THREAD

**Issue by ross-spencer** [archivematica/Issues#739]
**Expected behaviour**

pbzip2 compressed packages can be reingested. 

**Current behaviour**

* The workflow string [incomplete](https://github.com/artefactual/archivematica/blob/qa/1.x/src/MCPServer/lib/assets/workflow.json#L1056).
* This results in an empty `TRANSFORMALGORITHM`:
```xml
      <mets:file ID="pbzip-ae0b9327-1b7f-43f9-8e07-4be4830259c5.tar" GROUPID="Group-ae0b9327-1b7f-43f9-8e07-4be4830259c5" ADMID="amdSec_113644">
        <mets:FLocat xlink:href="/var/archivematica/sharedDirectory/www/AIPsStore/ae0b/9327/1b7f/43f9/8e07/4be4/8302/59c5/pbzip-ae0b9327-1b7f-43f9-8e07-4be4830259c5.tar.bz2" LOCTYPE="OTHER" OTHERLOCTYPE="SYSTEM"/>
        <mets:transformFile TRANSFORMTYPE="decompression" TRANSFORMORDER="1" TRANSFORMALGORITHM=""/>
      </mets:file>
```
* Which is going to hit this part of the code and result in a `KeyError` [here](https://github.com/artefactual/archivematica-storage-service/blob/946a513da208f96c87b00f0ae86ad011d7e9baac/storage_service/locations/models/package.py#L2165) where we look up an empty string. 

**Steps to reproduce**

Take a vanilla installation but set the config to compress the AIP as `Parallel BZip`. Once the AIP is stored. Do not change any config, and then choose to reingest. Approve the reingest and then watch for a failure during `Store AIP`. 

**Your environment (version of Archivematica, OS version, etc)**

`1.9.1` (`sandbox@artefactual`)

**Additional context**

A lot of errors are swallowed by the storage service, and so in the microservice (`Store the AIP`) you may only see:
```
500 Server Error: INTERNAL SERVER ERROR for url: https://sandbox.archivematica.org:8000/api/v2/file/51ce3420-c2c8-4b15-a15c-0e8edaab701b/Traceback (most recent call last):
  File "/opt/archivematica/archivematica/src/MCPClient/lib/job.py", line 102, in JobContext
    yield
  File "/opt/archivematica/archivematica/src/MCPClient/lib/clientScripts/store_aip.py", line 313, in call
    args.sip_uuid, args.sip_name, args.sip_type))
  File "/opt/archivematica/archivematica/src/MCPClient/lib/clientScripts/store_aip.py", line 212, in store_aip
    related_package_uuid)
  File "/opt/archivematica/archivematica/src/MCPClient/lib/clientScripts/store_aip.py", line 67, in _create_file
    agents=get_agents_from_db(uuid)
  File "/usr/lib/archivematica/archivematicaCommon/storageService.py", line 339, in create_file
    response.raise_for_status()
  File "/usr/share/archivematica/virtualenvs/archivematica-mcp-client/local/lib/python2.7/site-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 500 Server Error: INTERNAL SERVER ERROR for url: https://sandbox.archivematica.org:8000/api/v2/file/51ce3420-c2c8-4b15-a15c-0e8edaab701b/
```
In the storage service you should see this log output:
```
archivematica-storage-service_1  | ERROR     2019-06-12 10:34:54  locations.models.package:package:finish_reingest:2335:  Failed to extract valid compression algorithm from ""; does not match any of the following recognized options: 7z with bzip, 7z with lzma, 7z without compression, tar, tar bz2, tar gz
```

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by ross-spencer** [comment]
Hi @sromkey I have added `1.10` as this is resolved through ours' and Wellcome's work on `1.10` to add `gzip` compression to Archivematica. (So the PRs associated with this ticket). Have also placed it into `status: review`.

**Comment by sallain** [comment]
Set compression option to Parallel bzip2, created an AIP, reingested using the partial reingest option, and everything worked fine.
